spark/spark-tensorflow-connector/pom.xml

```diff
@@ -33,7 +33,7 @@
         <scala.test.version>2.2.6</scala.test.version>
         <maven.compiler.version>3.0</maven.compiler.version>
         <java.version>1.8</java.version>
-        <spark.version>2.3.0</spark.version>
+        <spark.version>2.4.0-SNAPSHOT</spark.version>
```

Comment:
I use 2.4.0-SNAPSHOT to verify Spark 2.4 compatibility. The code change should work with both 2.3 and 2.4. Before merge, I can revert the change here.